api/src/controllers/reviews.ts

```diff
@@ -95,7 +94,7 @@ router.get('/', async function (req, res) {
     courseID,
     professorID,
     userID,
-    _id: reviewID === undefined ? undefined : new ObjectId(reviewID),
+    _id: reviewID === undefined ? undefined : reviewID,
     verified: verified === undefined ? undefined : verified === 'true' ? true : false,
```

Comment:
I feel like this can be written better but it's not something that originally came up in the PR so I don't think it has to be addressed right now.